### ISSUE TITLE
Long queue times for dual mesh 30km-60km

### ISSUE THREAD

**Issue by jjguerrette** [NCAR/MPAS-Workflow#137]
These referenced lines show the current resource requests for dual-mesh 30km-60km 3denvar and 3dhybrid scenarios:
https://github.com/NCAR/MPAS-Workflow/blob/develop/scenarios/base/variational.yaml#L287-L289
https://github.com/NCAR/MPAS-Workflow/blob/develop/scenarios/base/variational.yaml#L294-L296

Currently the default behavior is to request 6 nodes, each with 32 PE and 109GB of memory.  The big memory nodes take a long time in the queue, even though this is the most cost-effective combination.  The single-precision benchmark set of observations is now below 240GB of total memory (could fit within 45GB per node) for some cycles, but maybe not all.  So users could try decreasing to 45GB (small memory nodes).  If many more observations are added beyond the benchmark experiment (increasing needed memory), and the long queue times are cumbersome for running your experiments, you can add one of the configurations below to your experiment scenario YAML.

```
variational:
  job:
    30km:
      60km:
        3denvar:
          nodes: 12
          PEPerNode: 16
          memory: 45
        3dhybrid:
          nodes: 12
          PEPerNode: 16
          memory: 45
```
OR
```
variational:
  job:
    30km:
      60km:
        3denvar:
          nodes: 8
          PEPerNode: 24
          memory: 45
        3dhybrid:
          nodes: 8
          PEPerNode: 24
          memory: 45
```

Note that the entire `job` node within the `variational` node can be copied to the existing `variational` node in your scenario YAML.  There is no need to create a second `variational` node in your scenario YAML.

@liujake @junmeiban @byoung-joo @metdyn @ibanos90

**Comment by junmeiban** [comment]
@jjguerrette , Many thanks for this information!

**Comment by jjguerrette** [comment]
No problem @junmeiban.  I updated the default resource requests in #137.  After using the `qhist` command from the CISL article Jake recommended, I saw that all of my recent `30km.60km` variational executions used less than 250GB of memory.  With 6 nodes, we have 270GB available on small memory nodes.  So I changed the default to use 45GB.  That may need to be increased with more observation types, and we can iterate on what the default values for `nodes`, `PEPerNode`, and `memory` ought to be.

I also tried a `30km.60km` 3denvar with 80 members on 12 nodes x 32 PEPerNode (384 PE total, double the default 192 PE).  That only reduced the wall-time by ~13%, with most of the savings in the ensemble covariance and localization multiplication (very scalable procedures).  The time it took to read the ensemble members increased 20%, and the observation I/O remained constant (`max pool size: 1`, known issue to be addressed in IODA). If we want to scale to larger processor counts for the same 3denvar and 3dhybrid simulations using 80-member ensembles, we might want to consider the `State::read` method as a potential bottleneck. We might consider using MPAS's functionality for dedicated I/O processors instead of using all available processors. @liujake @metdyn

For the details, see these two log files:
/glade/scratch/guerrett/pandac/guerrett_3denvar-60-60-iter_O30kmIE60km_B-SE80+RTPP70_12x32pe/CyclingDA/2018041500/run/mem001/jedi.log

/glade/scratch/guerrett/pandac/guerrett_3denvar-60-60-iter_O30kmIE60km_B-SE80+RTPP70_10AUG2022/CyclingDA/2018041500/run/mem001/jedi.log
